docs/content/stable/explore/going-beyond-sql/connection-mgr-ysql.md

```diff
@@ -81,11 +81,11 @@ The following table describes YB-TServer flags related to YSQL Connection Manage
 
 | flag | Description | Default |
 |:---- | :---------- | :------ |
-| enable_ysql_conn_mgr | Enables YSQL Connection Manager for the cluster. YB-TServer starts a YSQL Connection Manager process as a child process. | false |
+| enable_ysql_conn_mgr | Enable YSQL Connection Manager for the cluster. YB-TServer starts a YSQL Connection Manager process as a child process. | false |
 | enable_ysql_conn_mgr_stats | Enable statistics collection from YSQL Connection Manager. These statistics are displayed at the endpoint `<ip_address_of_cluster>:13000/connections`. | true |
-| ysql_conn_mgr_idle_time | Specifies the maximum idle time (in seconds) allowed for database connections created by YSQL Connection Manager. If a database connection remains idle without serving a client connection for a duration equal to, or exceeding this value, it is automatically closed by YSQL Connection Manager. | 60 |
+| ysql_conn_mgr_idle_time | Maximum idle time (in seconds) allowed for database connections created by YSQL Connection Manager. If a database connection remains idle without serving a client connection for a duration equal to, or exceeding this value, it is automatically closed by YSQL Connection Manager. | 60 |
 | ysql_conn_mgr_max_client_connections | Maximum number of concurrent client connections allowed. | 10000 |
-| ysql_conn_mgr_max_conns_per_db | Maximum number of concurrent database connections YSQL Connection Manager can create per pool. If set to zero, get max connections from pgconf file, 10% for control connection and 90% for global. | 0 |
+| ysql_conn_mgr_max_conns_per_db | Maximum number of concurrent database connections YSQL Connection Manager can create per pool. If set to zero, get maximum connections from the pgconf file, 10% for control connection and 90% for global. | 0 |
```

Comment:
"10% for control connection and 90% for global" - is this the default in the pgconf file, or what exactly

Reply:
The logic for the allocation of 10% for control connection and global connection is hardcoded in the code.